### PR TITLE
Update migrate-azure-analysis-services-to-powerbi-premium.md

### PR DESCRIPTION
**Pull request by ogetchie**
Updated to show Server properties are in PBI in public preview

### CODE REVIEW

**Comment by PRMerger14** [comment]
@ogetchie : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Jak-MS** [comment]
@peter-myers 

Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: powerbi-docs/guidance/migrate-azure-analysis-services-to-powerbi-premium.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ AAS provides the Analysis Services database engine for hosting data models, whic
 | [Enhanced refresh](/power-bi/connect-data/asynchronous-refresh), which allows any programming language to perform asynchronous dataset refreshes by using a REST API call | Yes | Yes |
 | [Backup and restore](../enterprise/service-premium-backup-restore-dataset.md) | Yes | Yes |
 | [Dataset workload settings](/power-bi/enterprise/service-admin-premium-workloads#datasets), which control Premium capacity workloads | No | Yes |
-| [Server properties](/analysis-services/server-properties/server-properties-in-analysis-services), which control Analysis Services server instance properties | Yes | No <sup>2</sup> |
+| [Server properties](/analysis-services/server-properties/server-properties-in-analysis-services), which control Analysis Services server instance properties | Yes | Yes <sup>2</sup> |
```
Comment (peter-myers) [inline powerbi-docs/guidance/migrate-azure-analysis-services-to-powerbi-premium.md:108]: The footnote "2" needs to be removed also. Remove " <sup>2</sup>".

**Review by peter-myers — CHANGES_REQUESTED** [review]
See my comment about removing the "2" superscript.

**Comment by maggiesMSFT** [comment]
Closing. Change is in https://github.com/MicrosoftDocs/powerbi-docs-pr/pull/8296

**Pull request closed without merging**